README.md

```diff
@@ -1,4 +1,5 @@
-### Welcome to the *Zend Framework 2.3* Release!
+![Logo](http://framework.zend.com/images/logos/ZendFramework-logo.png)
```

Comment:
@froschdesign should be fixed imo, but isn't a big deal. I just want to do it right if we introduce it.